### PR TITLE
Pauling release notes draft 2

### PR DESCRIPTION
**Pull request by wteouchicago**
- added feature and 508 changes
- updated known issues
- bug fixes 

__
SJ's changes will be in a separate PR.

### CODE REVIEW

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 1]
```diff
@@ -45,29 +45,116 @@
...
+    * Styling for the tool cards in the Analysis Center has been standardized. <!--PEAR-1957-->
+    * The search bar in the left panel within the __Clinical Data Analysis__ tool now remains fixed at the top of the page. <!--PEAR-1963-->
+    * The message "No data for this field" will only be displayed when information for a filter card has been loaded. <!--PEAR-2036-->
+    * Vertical alignment has been improved for tables that are displayed next to each other. <!--PEAR-2049--> 
+    * Filter panels located on the left side of the __Projects__, __Repository__, and __Mutation Frequency__ tools will now extend up to the height of the tables in the tools. <!--PEAR-2110-->
+    * Styling for survival plots has been improved for consistency. <!--PEAR-2176-->
+    * Download icons have been standardized. <!--PEAR-2186-->
+    * Text size has been increased for instructions in modals for selecting cohorts. <!--PEAR-2175-->
+* __File Summary Page__:
+    * The __Reference Genome__ section is no longer displayed in the the for files that have not been processed with the reference genome <!--PEAR-1967-->
```
Comment (wwysoc2) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:74]:
```suggestion
    * The __Reference Genome__ section is no longer displayed in the the for files that have not been processed with the reference genome. <!--PEAR-1967-->
```

**Review by wwysoc2 — APPROVED** [review]
Suggested one missing period.  Otherwise lgtm. 👍

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md (hunk 1 above)
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md]: fixed. also fixed other typos found.

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 2]
```diff
@@ -45,29 +45,116 @@
...
+    * The __Reference Genome__ section is no longer displayed for files that have not been processed with the reference genome. <!--PEAR-1967-->
+    * The Case ID column is now displayed by default in the Annotations table. Additionally, the Case UUID column is no longer displayed by default. <!--PEAR-1968-->
+    * Pagination has been added to the Read Groups table. <!--PEAR-2192-->
+    * Sample Type has been removed from the Associated Cases/Biospecimens table and replaced with Tissue Type and Tumor Descriptor. <!--PEAR-2042-->
+* __Repository__:
+    * Stability improvements have been added. <!--PEAR-2022-->
+    * The placement and design of the buttons to add custom filters and reset them have been updated. <!--PEAR-2059-->
+* __Clinical Data Analysis__:
+    * The y-axis of histograms will now only display integers for case counts. <!--PEAR-2087-->
+    * The rounding of numbers displayed in th etool has been improved. <!--PEAR-1040-->
```
Comment (clsackett) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:83]: typo: "th etool" --> "the tool"

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 3]
```diff
@@ -45,29 +45,116 @@
...
+* Removed "undefined" text in the __Survival Plot__ of __Mutation Frequency__ when no data is available in the Mutations tab, ensuring consistent messaging with the Genes tab. <!--PEAR-1524-->
+* Fixed an issue in __Set Operations__ where cohorts were not properly displayed after saving an "Unsaved_Cohort" during comparison, ensuring correct cohort selection and comparison behavior. <!--PEAR-2009-->
+* Users can now change the number of rows displayed in the table when selecting an existing cohort as the basis of a new cohort. <!--PEAR-2198-->
+* Fixed sorting functionality for "Submitted Gene Identifier" columns (Symbol, Ensembl ID, Entrez ID) in __Manage Sets__ to correctly sort by numbers/alphabet. <!--PEAR-2045-->
+* In the __Case Summary Page__, the issue with the Create Cohort button label case count in the Most Frequent Somatic Mutations table not matching the actual cohort case count has been resolved to ensure accurate numerator and denominator calculations. <!--PEAR-2142-->
+* Fixed issue where Y-axis labels on __Clinical Data Analysis__ histograms were cut off; labels are now fully visible. <!--PEAR-2055-->
```
Comment (clsackett) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:134]: Consider changing "Y-axis" to "y-axis" for consistency

**Review by clsackett — COMMENTED** [review]
One small typo and one minor change otherwise LGTM

**Comment by wteouchicago** [comment]
all feedback has been addressed. 🤞

**Review by clsackett — APPROVED** [review]

**Pull request merged**